### PR TITLE
doc: Remove explicit storage requirement from README.md

### PR DESCRIPTION
**Pull request by merland**
Similarly discussed and fixed in the following bitcoin.org issue: https://github.com/bitcoin-dot-org/bitcoin.org/pull/2716

### CODE REVIEW

**Comment by laanwj** [comment]
didn't even know there was another reference to the size here; there's this one in the release process
```
* Update [`BLOCK_CHAIN_SIZE`](/src/qt/intro.cpp) to the current size plus some overhead
```
maybe better to remove this one completely

File: doc/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
```
Comment (promag) [inline doc/README.md]: Could say that currently grows around 50GB/year?

Comment (laanwj) [reply doc/README.md]: I don't htink that's very useful here

**Comment by laanwj** [comment]
looks good to me now!

File: doc/README.md (hunk 1 above)
Comment (merland) [reply doc/README.md]: This doc is not a guide for running a node, but a more general text. As such, it does not need to be specific about the storage reqs, in my opinion. This passage would benefit from a general enough phrasing that won't need to be updated for a long time. Even the text "several hundred gigabytes" is not future proof, but that may be an ok tradeoff. 
If we find a good and independent reference source for the size further on, we can add a link to it then.

**Comment by jonasschnelli** [comment]
ACK 3a4757c2bba8cd59c08dc8b717e527115018c600

**Comment by promag** [comment]
ACK.

**Comment by donaloconnor** [comment]
ACK

**Review by kristapsk — CHANGES_REQUESTED** [review]

**Comment by fanquake** [comment]
ACK please squash your commits

**Comment by merland** [comment]
Thanks for your comment. Squashed 4 commits.

**Comment by fanquake** [comment]
utACK 5918204

**Comment by sipa** [comment]
ACK

**Pull request merged**